src/ioredis-cluster/getInfo.js

```diff
@@ -0,0 +1,22 @@
+const Redis = require('ioredis')
+const redisInfo = require('redis-info')
+const hash = require('object-hash')
+
+const redisInfoCache = {}
+module.exports = async function getInfo(server, options={}){
+    const {
+      cache = true,
+      force = false,
+    } = options
+    const id = cache ? hash(server) : null
+    if(cache && !force && redisInfoCache[id]){
```

Comment:
it's used only when you do
```
await new Redis(server, {autoDetectCluster: true})
```
it's not used in redis-ui, it's an optimization in case of dropin replacement of ioredis by ioredis-cluster, I will leave this feature when I will decouple as an independent library